tests/spdx/parser/jsonlikedict/test_error_message.py

```diff
@@ -19,13 +24,13 @@ def test_error_message():
         err.value.get_messages(),
         [
             'Error while constructing Package: [\'SetterError Package: argument "name" '
-            "(None) is not an instance of str', 'SetterError Package: argument "
-            '"download_location" (int) did not match any element in the union:\\n  str: '
-            "is not an instance of str\\n  "
-            "spdx_tools.spdx.model.spdx_no_assertion.SpdxNoAssertion: is not an instance "
-            "of spdx_tools.spdx.model.spdx_no_assertion.SpdxNoAssertion\\n  "
-            "spdx_tools.spdx.model.spdx_none.SpdxNone: is not an instance of "
-            "spdx_tools.spdx.model.spdx_none.SpdxNone', 'SetterError Package: item 1 of "
-            'argument "attribution_texts" (list) is not an instance of str\']'
+            "(None) is not an instance of str: None', 'SetterError Package: argument "
+            '"download_location" (int) did not match any element in the union: '
+            "[\\'str\\', \\'spdx_tools.spdx.model.spdx_no_assertion.SpdxNoAssertion\\', "
```

Comment:
I'd find it better not to have quotation marks around the types as this leads to escaped marks in the error output, which does not look good :)

Reply:
done